presidio-analyzer/requirements.txt

```diff
@@ -0,0 +1,35 @@
+argcomplete==1.11.1
```

Comment:
We'd rather not have a Pipfile + requirements.txt. We moved away from requirements.txt because the installation here is challenging enough even without dependency conflicts

Reply:
I can remove this.  I am using anaconda for testing because I've read in the past that the pytorch wheels don't have as many optimizations as the pytorch on anaconda.  Also pipenv doesn't play very nicely with anaconda, so I duplicated what was in the `Pipfile.lock` with a simple jq script.  But I can just have all this in my own version of the repo.

Reply:
I think it would be best. Either we move completely to Conda or we stay with Pipenv. Having both is confusing and difficult to maintain. Honestly Pipenv isn't a perfect tool (especially since it's not actively maintained anymore) but we have a setup which is working here and it would require some development to replace it with Conda.

Reply:
I deleted the `requirements.txt` file